sdks/python/apache_beam/coders/typecoders.py

```diff
@@ -70,6 +70,8 @@ def MakeXyzs(v):
 from apache_beam.coders import coders
 from apache_beam.typehints import typehints
 
+__all__ = ['CoderRegistry']
```

Comment:
Actually, I think registry is the only public thing here. But remove this; I'll take care of it with my coder changes.